web/wp-content/themes/bumblebee/footer.php

```diff
@@ -78,18 +78,7 @@
 				</div>
 			</div>
 			<div class="pure-u-1 pure-u-md-2-5 right-foot">
-				<div class="newsletter">
-					<h3><?php echo esc_html( get_theme_mod( 'bumblebee_footer_nl_heading_text' ) ); ?></h3>
-					<form action="<?php echo esc_url( get_site_url() ); ?>/newslettersignuppage/" method="post" data-analytics-metrics='{"name":"newsletter signup","module":"newsletter signup","position":"footer"}' >
-						<input type="text" id="email" placeholder="Email Address"></input>
-						<button type="submit" id="subscribe">Sign Up</button>
-					</form>
-				</div>
-				<div class="diyu-logo">
-					<a data-analytics-metrics='{"name":"Subscribe link","module":"footer","position":"magazine subscription"}' href="<?php echo esc_html( get_theme_mod( 'bumblebee_footer_nl_subscribe_url' ) ); ?>" target="_blank" rel="noopener noreferrer">
-						<img src="<?php echo esc_html( get_theme_mod( 'bumblebee_footer_nl_subscribe_image' ) ); ?>" alt="" style="width:<?php echo esc_html( get_theme_mod( 'bumblebee_footer_nl_subscribe_image_width' ) ); ?>px"></img>
-					</a>
-				</div>
+				<?php newsletter_module(); ?>
```

Comment:
Is it worth having a ticket to make this a widget?

Reply:
This was added to the customizer in https://github.com/fariasf/drdt-test/pull/82/